src/displayapp/screens/Navigation.cpp

```diff
@@ -24,6 +24,106 @@ using namespace Pinetime::Applications::Screens;
 
 LV_FONT_DECLARE(lv_font_navi_80)
 
+namespace {
+  static constexpr std::array<std::pair<const char*, const char*>, 86> m_iconMap = {{
```

Comment:
Somewhat a matter of style preferences but anything declared in an anonymous namespace is automatically `static` so the extra static here is redundant.